docs/08-MultiPageDashApp.md

```diff
@@ -251,7 +251,45 @@ However, we recommend splitting out the page layouts into their own files in `pa
 
 ***
 
+**Query Strings**
 
+It's possible to pass query strings from the url to a layout function.
+For example:
+```python
+import dash
+
+dash.register_page(__name__, path='/dashboard')
+
+def layout(velocity=0, **other_unknown_query_strings):
+    return dash.html.Div([
+        dash.dcc.Input(id='velocity', value=velocity)
+    ])
+
+```
+![image](https://user-images.githubusercontent.com/72614349/146809878-3592c173-9764-4653-89aa-21094288ca0a.png)
+
+**Path Variable**
+
+Another way to pass variables to the layout is to use the `path_template` parameter in  `dash.register_page`.  You can
+define which segments of the path are variables by marking them like this: `<variable_name>`. The layout function then receives the `<variable_name>` as a keyword argument.
+
+For example, if `path_template= "/asset/<asset_id>"`, and the url is "/assets/a100", then the layout
+will receive `{"asset_id":"a100"}`.  Here is an example with two variables in the path:
+
+```python
+import dash
+
+dash.register_page(
+    __name__,
+    path_template="/asset/<asset_id>/department/<dept_id>",
```

Comment:
Any reason not to just overload path= so we could just have one argument?

Reply:
There needs to be a way to specify the path_template which defines the position of the variables and the names.  There also needs to be a way to specify a valid staring path.  I think it makes sense to have two props for this.